core/go/internal/txmgr/transaction_submission.go

```diff
@@ -574,6 +574,19 @@ func (tm *txManager) resolveNewTransaction(ctx context.Context, dbTX persistence
 	bypassFromCheck := submitMode == pldapi.SubmitModePrepare || /* no checking on from for prepare */
 		(submitMode == pldapi.SubmitModeCall && tx.From == "") /* call is allowed no sender */
 	if !bypassFromCheck {
+		if strings.HasPrefix(tx.From, "verifier:") {
+			addr := strings.TrimPrefix(tx.From, "verifier:")
+			if _, err := pldtypes.ParseEthAddress(addr); err != nil {
```

Comment:
So just to be clear this means we'll only support Ethereum Address verifiers at the end of this PR.